pkg/logql/engine.go

```diff
@@ -448,7 +448,7 @@ func (q *query) evalLiteral(_ context.Context, expr *syntax.LiteralExpr) (promql
 		return nil, err
 	}
 	s := promql.Scalar{
-		T: q.params.Start().UnixNano() / int64(time.Millisecond),
```

Comment:
Checkout line 468. I think that one is wron. And then there's 474 that just uses `UnixMilli()` :thinking: